testsuite/E30.py

```diff
@@ -118,6 +118,28 @@ def a():
     a()
 #:
 
+#: E306:3:5
```

Comment:
This looks great to me, but could you maybe add some `#: Okay` test cases that verify the desired behavior?